api/server/router/local/image.go

```diff
@@ -433,6 +448,10 @@ func (s *router) postBuild(ctx context.Context, w http.ResponseWriter, r *http.R
 		}
 	}
 
+	if !buildConfig.Verbose {
```

Comment:
Like on the client, let's add a comment here so people understand that when -q is specified there should be no output except the image ID, so we need to send it here.